ansible/playbooks/start-redpanda.yml

```diff
@@ -7,44 +7,63 @@
       state: directory
 
   - name: configure redpanda
-    notify:
-      - restart redpanda-tuner
-      - restart redpanda
     vars:
       use_public_ips: "{{ advertise_public_ips | d() | bool }}"
     shell: |
       set -e
-      rpk config set cluster_id 'redpanda'
-      rpk config set organization 'redpanda-test'
-      rpk config set redpanda.advertised_kafka_api '{
+      sudo -u redpanda rpk redpanda config set cluster_id 'redpanda'
```

Comment:
All `rpk` commands are updated to the new syntax to remove the deprecated warnings from the logs. Also the commands are now being ran by the redpanda user (since that is the owner of the files being modified).

Reply:
Nice, thanks!